system/lib/wasmfs/wasmfs.cpp

```diff
@@ -63,6 +66,8 @@ void WasmFS::preloadFiles() {
     return;
   }
 
+  auto rootBackend = wasmFS.getRootDirectory()->locked().getBackend();
```

Comment:
Is this meant to make the code more future-proof? That seems reasonable perhaps. But it seems simpler to assign to `rootBackend` up at the top of the function when we create it.

Reply:
Yeah I was aiming to make this future-proof instead of hard-coding `MemoryFile` here.